lib/coinbase/wallet.rb

```diff
@@ -126,16 +126,19 @@ def seed=(seed)
     # Creates a new Address in the Wallet.
     # @return [Address] The new Address
     def create_address
-      key = derive_key
-      attestation = create_attestation(key)
-      public_key = key.public_key.compressed.unpack1('H*')
+      opts = { create_address_request: {} }
```

Comment:
Can we add a context for the `#create_address` spec when we do not use server signing?